saleor/product/models.py

```diff
@@ -185,6 +185,11 @@ class ProductVariant(models.Model):
         Product, related_name='variants', on_delete=models.CASCADE)
     attributes = HStoreField(default={}, blank=True)
     images = models.ManyToManyField('ProductImage', through='VariantImage')
+    handle_stock = models.BooleanField(
+        default=True,
+        help_text=pgettext_lazy(
+            'product variant handle stock field help text',
+            'Allow or not to automatically handle stock units from orders'))
```

Comment:
We tend to keep help texts in forms rather than in models. Also, if we changed `handle_stock` to `track_inventory` we would have to rephrase it to something like: "Automatically track this product's inventory".